OneSignalSDK/unittest/src/test/java/com/test/onesignal/RESTClientRunner.java

```diff
@@ -92,23 +99,38 @@ public void testRESTClientFallbackTimeout() throws Exception {
 
    @Test
    public void SDKHeaderIsIncludedInGetCalls() throws Exception {
-      OneSignalRestClient.getSync("URL", null, null);
+      new Thread(new Runnable() {
+         @Override
+         public void run() {
+            OneSignalRestClient.getSync("URL", null, null);
```

Comment:
This could be changed to just `OneSignalRestClient.get("URL", null, null);` which is asynchronous. This would simplify this.
Or does this create any issues with the tests getting stuck?